src/hooks/useObservations.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect } from 'react'
+import { useImmer } from 'use-immer'
+import { loadFromLocal, saveToLocal } from '../utils/localStorage'
+
+export default function useObservations(selectedDayString) {
+  const observationTypes = [
+    { name: 'Größe', type: 'size', unit: 'cm', format: '134' },
+    { name: 'Gewicht', type: 'weight', unit: 'kg', format: '25,95 ' },
+    { name: 'Temperatur', type: 'temperature', unit: '°C', format: '36,7' },
+    {
+      name: 'Blutdruck',
+      type: 'bloodpressure',
+      unit: 'mmHg',
+      format: '104/57',
+    },
+    { name: 'FEV1', type: 'fev1', unit: 'l/s', format: '1,26' },
+    { name: 'Blutzucker', type: 'bloodsugar', unit: 'mmol/l', format: '6,7' },
+    { name: 'Urin', type: 'urine', unit: '' },
+    { name: 'Notizen', type: 'notes', unit: '' },
+  ]
```

Comment:
Maybe save as a json file and import that file as obvservationTypes

Reply:
This is what I did before and then I changed it... the advantage would also be that I don't have to pass it through.
But I like it that it is all in one place. I have to think about that 😄